packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -10,221 +10,21 @@
  * governing permissions and limitations under the License.
  */
 
-import {AriaLabelingProps, DOMProps, DOMRef, DropTarget, FocusableElement, FocusableRef, SpectrumSelectionProps, StyleProps} from '@react-types/shared';
```

Comment:
See https://github.com/adobe/react-spectrum/pull/4641/commits/dfcdea154620b418190b4d44193aba7a3ac89e03 for a better diff of TableView changes. Hard to see here due to the renaming of the files